api/helpers/unlock-game.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  friendlyName: 'Unlock a game',
+
+  description:
+    'Unlock a game using the previously required lock UUID. Will unset lock and lockedAt columns for the game matching the current lock.',
+
+  inputs: {
+    lock: {
+      type: 'string',
+      descritiption: 'The lock UUID to unset/release ',
+      example: '921a780a-514e-4086-8c9c-fccc2a8d26d8',
+      required: true,
+    },
+  },
+
+  fn: async ({ lock }, exits) => {
+    await Game.updateOne({ lock }).set({lock: null, lockedAt: null});
+    return exits.success();
```

Comment:
Interestingly, I don't think we care about whether the unlock write operation successfully writes to the db because if someone else has locked the game, we just let them have it until the lock times out. That's why this returns void. That might be worth a comment

Reply:
Does it matter with a distinction between when it times out vs. when the operation succeeds from the user who locked it? It sounds like:

- a user who locks it can unlocks it
- a user who did not lock it can not unlock it
- It can timeout and unlock on its own (is this part of the check of a new request or does a separate job run to unlock?)

Reply:
Correct. The timeout is only folded into the check for a new attempt to lock; there is no separate job to clear it later. 
When you try to unlock a game, either:
1. You succeed and the game is now unlocked
2. You fail because your lock expired and someone else locked the game before you unlocked
3. An unexpected error e.g. server loses connection to the database causes a failure.

3) should be an error and already should cause the method to error when the `Game.updateOne()` call errors. I don't _think_ the caller of `unlock-game` needs to care about the difference between 1) and 2) because either way, the request that has called `unlock-game` is no longer locking that game and now it's up to whatever other requests interact with the game later on to sort out who's locking the game (if anyone). So I think it's still reasonable for this method to resolve void as long as it doesn't hit an adapter error, whether the unlock write operation is successful or not